pages/blog/blog-posts.css

```diff
@@ -1,11 +1,62 @@
 .blog-post-wrapper {
     position: relative;
     top: 90px;
+
+    /* means our content will not become super wide on larger screens*/
+    max-width: 1200px;
+
+    /*means our 'main' element will always be centered, no matter the device */
+    /**/
+    margin: 0 auto;
 }
 
 .blog_post_header {
     display: flex;
-    flex-direction: row;
```

Comment:
'flex-direction: row' does the same thing as 'display: flex' , so I've removed it 😄